File: src/main/java/maven/example/com/receiving/system/HWID.java
package maven.example.com.receiving.system;

import maven.example.com.receiving.tools.Receipt;
import maven.example.com.tools.data.Data;
import maven.example.com.tools.data.TypeData;
import maven.example.com.tools.rest.Hwid;

public class HWID extends Receipt {
    private static final TypeData TYPE = TypeData.SYSTEM_HWID;

    public HWID() {
        super(TYPE);
    }

    @Override
    public Data getData() {
        return new Data(TYPE, Hwid.getHWID());
    }
}

File: src/main/java/maven/example/com/receiving/system/HWID.kt
package maven.example.com.receiving.system

import maven.example.com.receiving.utility.Receipt
import maven.example.com.utility.data.Data
import maven.example.com.utility.data.TypeData
import maven.example.com.utility.rest.Hwid

class HWID : Receipt(TYPE) {

    override val data: Data?
        get() = Data(TYPE, Hwid.getHwid())

    companion object {
        private val TYPE = TypeData.SYSTEM_HWID
    }
}
